apps/design/frontend/src/export_screen.tsx

```diff
@@ -25,6 +30,20 @@ export function ExportScreen(): JSX.Element {
     );
   }
 
+  function onPressExportTestDecks() {
+    exportTestDecksMutation.mutate(
+      { electionId },
+      {
+        onSuccess: ({ zipContents, electionHash }) => {
+          fileDownload(
```

Comment:
I'm wondering whether this should be done as a standard browser file download, i.e. an endpoint that streams an `application/octet-stream` rather than using grout for this, which requires that we hold the entire thing in memory both in the frontend and the backend. I'm not sure it's worth it to do now, but perhaps as a follow-up task.

Reply:
Yeah, I think once we move all of these exports to the background worker (which is the next planned change), that will become necessary. I'm imagining we might upload the files to s3 and then have a direct download link from there. TBD!